### PR TITLE
doc: added release notes for v1.8 draft new

### PR DESCRIPTION
**Pull request by Shilpashree3lc-zz**
Draft of Zephyr 1.8 release notes which contains some interesting
changes on Kernel, Architecture, Boards, Drivers and Sensors,
Networking, Bluetooth, documentation, and jira items.etc.

Signed-off-by: LC Shilpashree shilpashree.lc@intel.com

### CODE REVIEW

**Comment by dbkinder** [comment]
This PR should only have one file being committed (doc/release-notes-1.8.rst)?

File: doc/release-notes-1.8.rst [hunk 1]
```diff
@@ -95,20 +190,27 @@ JIRA Related Items
...
+* :jira:`ZEP-634` - document IRQ handling internals
 * :jira:`ZEP-638` - feature to consider: flag missing functionality at build time when possible
 * :jira:`ZEP-720` - Add MAX30101 heart rate sensor driver
 * :jira:`ZEP-828` - IPv6 - Multicast Join/Leave Support
 * :jira:`ZEP-843` - Unified assert/unrecoverable error infrastructure
 * :jira:`ZEP-888` - 802.15.4 - Security support
 * :jira:`ZEP-932` - Adapt kernel sample & test projects
 * :jira:`ZEP-948` - Revisit the timeslicing algorithm
-* :jira:`ZEP-973` - Remove deprecated API related to device PM, DEVICE\_ and * SYS\_* macros
+* :jira:`ZEP-973` - Remove deprecated API related to device PM, DEVICE_ and * SYS_* macros
```
Comment (dbkinder) [inline doc/release-notes-1.8.rst]: DEVICE\_ and * SYS\_*  (need to escape the _ to keep Sphinx from thinking these are label reference links)

**Review by nashif — CHANGES_REQUESTED** [review]
Please the PR and make sure you are changing the last version of the release notes available in the branch, we received a few updates already.

**Comment by nashif** [comment]
superseded by #491

**Pull request closed without merging**